packages/ui/views/account/friends/search-friend/index.tsx

```diff
@@ -145,7 +143,7 @@ export default class SearchFriend extends Component<Props, State> {
       { hasSearchTerm &&
       <View style={friendStyle.searchList}>
         <Loading context={loadingContext} />
-        {hasSearchTerm && matches.length === 0 ? <Text style={[style.emptyState, {paddingLeft: 30}]}>{noMatches}</Text> : null}
+        {hasSearchTerm && matches.length === 0 ? <Text style={[style.emptyState, {paddingLeft: 30, paddingTop:30}]}>{noMatches}</Text> : null}
```

Comment:
Note my comment that `emptyState` does not exist for style = "form"

Reply:
![screen shot 2018-08-02 at 1 40 07 pm](https://user-images.githubusercontent.com/1081476/43609722-a10605c4-9659-11e8-913b-27f6e087ac8d.png)